### PR TITLE
Update Hebrew Translations

### PR DESCRIPTION
**Pull request by wgranger**
This PR updates Hebrew translations as identified in #83. One issue of that PR still remains, and some Arabic translations are still missing.

### CODE REVIEW

File: src/components/SelectedAnnotation.jsx [hunk 1]
```diff
@@ -192,12 +192,12 @@ class SelectedAnnotation extends React.Component {
   closePrompt() {
     this.props.dispatch(togglePopup(
       <QuestionPrompt
-        confirm="Yes, close without saving"
-        deny="No, continue transcribing"
+        confirm={this.props.translate('closeAnnotation.confirm')}
```
Comment (eatyourgreens) [inline src/components/SelectedAnnotation.jsx]: Maybe `const { translate } = this.props;` at the top of this function, to save some repetition in these props.

File: src/components/Home.jsx [hunk 2]
```diff
@@ -49,6 +49,9 @@ const Home = ({ currentLanguage, dispatch, history, rtl, translate }) => {
         <div>
           <ZooniverseLogotype width="100px" />
           <h1>{translate('topNav.site')}</h1>
+          {language !== LANGUAGES.ENGLISH && (
+            <h1>Scribes of the Cairo Geniza</h1>
```
Comment (eatyourgreens) [inline src/components/Home.jsx]: I'm not sure what's happening here. Does this add a second `h1` if the language isn't English? Documents should really only have one top-level heading.

Comment (wgranger) [reply src/components/Home.jsx]: The research team wanted the English variant kept when switching to Arabic and Hebrew. I can change it to `h2`, as that would probably make more sense.

File: src/components/SelectedAnnotation.jsx (hunk 1 above)
Comment (wgranger) [reply src/components/SelectedAnnotation.jsx]: Ah, good call. This will help readability.

**Pull request merged**